src/odemis/gui/model/__init__.py

```diff
@@ -1042,6 +1045,12 @@ def __init__(self, main):
         # For dichotomic mode
         self.dicho_seq = model.ListVA()  # list of 4 enumerated for each corner
 
+class EnzelAlignGUIData(ActuatorGUIData):
+    def __init__(self, main):
+        ActuatorGUIData.__init__(self, main)
+        self.viewLayout.value = VIEW_LAYOUT_VERTICAL
```

Comment:
Here, you can redefine it, with `self.viewLayout = model.IntEnumerated(VIEW_LAYOUT_22, choices={VIEW_LAYOUT_ONE, VIEW_LAYOUT_VERTICAL})` 
(Not sure if VIEW_LAYOUT_ONE is even possible, in which case, drop it).